tests/xtimer_usleep/tests/01-run.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Unrelated, because I have the same problem on master, but I can't execute this test specifically (`tests/lwip` works e.g.) because the `# -*- coding: utf-8 -*-` line isn't set in the `testrunner` module. Weird, but will fix.

Reply:
(on a system where `python` maps to `python2` ;-))